docs/install/tar.md

```diff
@@ -13,22 +13,38 @@ The tarball supports CentOS 7, Amazon Linux 2, Ubuntu 18.04, and most other Linu
 
 1. Download the tarball:
 
+   x64:
    ```bash
-   curl https://d3g5vo6xdbdb9a.cloudfront.net/tarball/opendistro-elasticsearch/opendistroforelasticsearch-{{site.odfe_version}}.tar.gz -o opendistroforelasticsearch-{{site.odfe_version}}.tar.gz
+   curl https://d3g5vo6xdbdb9a.cloudfront.net/tarball/opendistro-elasticsearch/opendistroforelasticsearch-{{site.odfe_version}}-x64.tar.gz -o opendistroforelasticsearch-{{site.odfe_version}}-x64.tar.gz
```

Comment:
All these tarball you miss the `linux` platform in the link.

Reply:
Made the change.